### ISSUE TITLE
indent bug with regexp

### ISSUE THREAD

**Issue by futurist** [mooz/js2-mode#382]
Think below code, in `js2-mode`, indent as below:

``` js
var re = /some value/
  str.match(re)
```

But below code is indent right:

``` js
var re = /some value/i
str.match(re)
```

Is this a bug?

**Comment by dgutov** [comment]
It's a bug in either `js--continued-expression-p` or `js--looking-at-operator-p`. Please report it to Emacs.

**Comment by dgutov** [comment]
Should be fixed in both indentation engines now. The only ones who won't see the fix are the users of Emacs 25.1 (sorry).

**Comment by futurist** [comment]
Thank you for the job!
